governance/delegates/0x00dAeC2C2A6a3FCc66b02E38B7e56dCDFa9347a1/profile.md

```diff
@@ -0,0 +1,60 @@
+**Key Info**
```

Comment:
Planet, this supports markdown, same as the forum. You might want to use # for headers instead of just bold text.